tests/transforms/test_qcut.py

```diff
@@ -1959,6 +1962,160 @@ def test_mc_state_prep(self):
                 assert op.wires == exp_op.wires
 
 
+class TestMCPostprocessing:
+    """
+    Tests that the postprocessing for circuits containing sample measurements
+    gives the correct results.
+    """
+
+    @pytest.mark.parametrize("interface", ["autograd.numpy", "tensorflow", "torch", "jax.numpy"])
+    def test_sample_postprocess(self, interface):
+        """
+        Tests that the postprocessing for the generic sampling case gives the
+        correct result
+        """
+        lib = pytest.importorskip(interface)
+        fragment_tapes = [frag0, frag1]
+
+        communication_graph = MultiDiGraph(frag_edge_data)
+        shots = 3
+        fragment_configurations, settings = qml.transforms.qcut.expand_fragment_tapes_mc(
+            fragment_tapes, communication_graph, shots
+        )
```

Comment:
```suggestion
```
Do we need this?

Reply:
BTW, not sure what IDE you are using, but on PyCharm this is easy to see with unused variables shown in a faded grey colour:
![image](https://user-images.githubusercontent.com/49409390/160293291-e193de30-33d0-4073-9d79-4ba711f4f07c.png)

Maybe something that can be enabled?